Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN git reset --hard HEAD && git status
 # because we are guaranteed only one version of Go
 # used to build protoc-gen-go
 RUN go generate
+RUN git checkout ssf/sample.pb.go
```

Comment:
We actually don't want to do this, because this bypasses the check we have (to ensure that we never commit a protobuf definition that's out of sync with the corresponding Go file).